src/app/right-hand-panel.js

```diff
@@ -13,20 +31,47 @@ import 'bootstrap';
 @customElement('right-hand-panel')
 @inject(Context, Element)
 export class RightHandPanel {
+
+    tabs = [
```

Comment:
actually I hit the problem last night trying to sort the tab-content and wanted to discuss that today with you
I was using ``<compose>`` and had some "strange" side-effects
Let's discuss later on